wrapper/src/main/java/software/amazon/jdbc/plugin/cache/CacheConnection.java

```diff
@@ -191,6 +262,30 @@ public CacheConnection(final Properties properties) {
     } else {
       this.credentialsProvider = null;
     }
+    // Initialize the message digest
+    if (msgHashDigest == null) {
+      try {
+        msgHashDigest = MessageDigest.getInstance("SHA-384");
+      } catch (NoSuchAlgorithmException e) {
+        throw new RuntimeException("SHA-384 not supported", e);
+      }
+    }
+    // Register cluster with CacheMonitor
+    if (initializeCacheMonitor) {
+      CacheMonitor.registerCluster(properties, telemetryFactory, this.cacheRwServerAddr, this.cacheRoServerAddr,
+          this.useSSL, this.cacheConnectionTimeout, this.iamAuthEnabled, this.credentialsProvider,
+          this.cacheIamRegion, this.cacheName, this.cacheUsername, this.cachePassword
+      );
+    }
+  }
```

Comment:
Same question here. Do we need to register this cache cluster to cache monitor if we don't run cache operations against it?